docs/guides/wifi.md

```diff
@@ -13,57 +13,66 @@ Refer to the "Updating Kernel" section on your distro's FAQ for instructions if
 - [Manjaro](https://wiki.t2linux.org/distributions/manjaro/faq/#updating-kernel)
 - [Ubuntu](https://wiki.t2linux.org/distributions/ubuntu/faq/#updating-kernel)
 
-## Getting the firmware
+## Setting up
 
-In order to get wifi running on Linux, we need to get the firmware from macOS and copy it to Linux with appropriate changes. Thanks to Asahi Linux, who has made the process much simpler. You may follow the steps below to get the firmware and copy it to Linux properly.
+[This gist](https://gist.github.com/AdityaGarg8/94c6a3291455d5f8b7567ed9fb0b651e) has a script which can help you set up Wi-Fi. Follow the instructions below to use this script :-
 
-1. Boot into macOS.
+### On macOS
 
-2. Open the terminal and run :-
+1. Click [here](https://gist.githubusercontent.com/AdityaGarg8/94c6a3291455d5f8b7567ed9fb0b651e/raw/wifi.sh) to download the script.
+2. Boot into macOS.
+3. Run this script there.
+4. The script will ask you to run some commands or the script itself in Linux after it gets executed successfully.
+5. Boot into Linux.
 
-   ```sh
-   cp -r /usr/share/firmware/wifi ~/Desktop
-   curl -L https://github.com/AsahiLinux/asahi-installer/archive/refs/heads/main.tar.gz > ~/Desktop/asahi-installer-main.tar.gz
-   ```
+### On Linux
 
-   This shall make a folder named **wifi** and a zip archive named **asahi-installer-main.tar.gz** on the macOS Desktop.
+You have two options here:
 
-3. Copy the **wifi** folder and the **asahi-installer-main.tar.gz** archive to an external drive or a partition which is formatted to a file system readable by Linux. You can also use your EFI partition by mounting it using `sudo diskutil mount disk0s1`.
+- The first is to either copy this script to Linux via a USB, download it if you have a wired internet connection, or use some other method to get it to Linux. You can then run the script again from Linux and it will finish setting up Wi-Fi.
 
-4. Boot into Linux.
+- The second method is to run the commands which the script asked you to run on macOS. The commands in most cases should be the following, though you have to stick to the commands asked by the script, in case they turn out to be different.
```

Comment:
This is quite confusing and could lead readers to think that the entire code block can't be trusted. It'd be better to tell them that "wifi.sh" on the last line could have a different name if they renamed the script on macOS, so they know what could need changing, and why it would need changing.

Reply:
d1202ac check